### ISSUE TITLE
fulldps when minion skill is main, in tree node power missing: POB-item-tester cannot weight in

### ISSUE THREAD

**Issue by redrios** [PathOfBuildingCommunity/PathOfBuilding#4627]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

I think this already has been reported, plz correct me anyways, or at least I saw a conversation about the node power when main skill is minion and defensive nodes (for character, life, es) weren't counting, one has a workaround, changing main skills, disabling fulldps check, etc, to make the tree node power the character not the minion. I'm not a programmer but i guess this issue is connected to the ITEM TESTER loading items fine, and comparing fine items copied, in terms of DPS and minion life etc, but it cannot generate power points for the chart or list of mods weighting for a auto-generated search, I guess it has to do with the values being copied from somewhere akin to the power nodes showing differences when hovering over them...

### What is the actual behaviour/value?

gives 0 weights

### How to reproduce the issue

https://github.com/VolatilePulse/PoB-Item-Tester

https://xanthics.github.io/PoE_Weighted_Search/?vals=6&FrenzyCount=3&Flags=Leeching,OneHandedWeapon,Multicastable,enemyIsBoss,Spell,Dagger,Totemable,feedingFrenzyFeedingFrenzyActive,Mineable,Duration,UsedMinionSkillRecently,raiseSpectreEnableBuffs,minionsUseFrenzyCharges,CreatesMinion,Cooldown,Trappable,raiseSpectreEnableCurses,Minion,Shield,&Skill=&Character=3%2E18%20necro

my petty necro

### Character build code

```shell
eNrdfftzm8iW8M9XfwXlrbv1fRXHoR80kJu5W37bEzvx2HnMzNZWqoHGIkGgAHLsmZr_fU93gwQSICQ72aq5typjiXNO9zl93nS3Xv3X_SQ27kSWR2ny0w7aM3cMkfhpECW3P-28f3fy3Nn5r3-PXl3xYvw2PJhFsXzy79E_Xqm_jVjcifinHdvdMQqe3YriQ0WKfAJSU54UY5Eml_xzmp2mwU87b9JE7BgeT4KoqD75Mc_zN3wiftr5GBX-eMfguS-S4HDx_RvhZ-mEJ77IdowJj5Kb1P8iitMsnU1h-B3jLhLfLtMAQM8vr95ev6uNHSX1sWHu_3h1FfMHkd0UvDBy-OennX2_iO4EgMLcL6JJVABFHs-AHDJ3XnTggNj4rTirAXfC3kyFCOZg9h6lXZBXmTgOQ6Hmc5hFxeFYcr1-hE1hL2dxEU3jSAq0YnXP6sJ4lxY8Prq6WU9bQ6YDZAJrPT6IQS6D6Ero89skKsRg8Ks0ytNko1kPAj6cxTFYwTDYdOJFyUAeL3nCD9N8ITuKu0AvolA0QDuJHt8Mg7sGXR4GKad5JTIw0WIYgpzsRgjlCDfCT8ENbDLGJig1PrYbbAvM45ttELYY6KZYGDamqHPVxdc6JOrWuCNxvwCzrR6CDUjKuiDPk5qiE6eHYB0S983wLi3AgQ809-OzqzkksS27E_pq_JBHPo8v-X00mU3A4b_jX8RiGMulZo-q3I6LBNxFFzY1idvJ0kmUia0QD9M42A5xzNO8C9NBprPnYGJhlyLk9JnHYsFs2-4Ji_5LCX2e-HMExvrovk8ykYvsrhZO-0aQKNdgNjJWe7EYjLMYpjS_WkJg9g92WxMYtuw9G-TVh3EhhD8-hZzmmhdimNdaTMZyaK9wJXRduJCp9dBtES6yzGEYy3Jy9yhzq__bfTSaImN0D_VBbyiujzwLBnjZRGS3DzfjSMQ11h1sD0Eo9evhkE8H4aqFqROoL5DrDBuyLrJBbG0ot-M7ntd9KWVr-NHwdVa6FUdA2gcIgRiasV5l6WeZE8eboe1nk3RWi4M2tc1eLjRCnQlK1oWFIz6B1OlaBDO_EX2cTsyDGKqXJR66fQRMLI7bUHCnLe8XBfe_HKXB7WBRqVE2x7iZTafgAeTSD8WTMe1a5FEti-lOJxbAb8HE6haGUV_sGzzAAnh5AIbWxvTBoyxhLA9lOb3heGmY53gA9PIQ3YZeVZuX4MQm4MFVrQq19ELSe91J3AnUQvXaBiFsMrSHbNO0EaokWELdfIni2Mjlv1c8kyo0m0ygKA-Mmy_gEoo0MQ55XojspTEXmLEw_R0jB-P0YZSdSgUV_hw7XzONVRNOv4GUxrJzkQ9wQjVoyJAWptspnEwkfzws0x8GPmiA4ySYZdLqBvOwjDFomAHdEf1wtTuinWPNy1k23XNtyiAzsrpwm90SsoeeAHKlg9GuIh14Lf2MFcjWfsbAUdq7Gxsgr_Y6NuSvrUGxAtzWzRg4TqMmIBbCfYBLSTSie6bD-hDakxtKnT1MKXNM7FDaqRrtyZ_ZPcX1mVgvSn8m9i6agDfL8yNecOOz-Cbidw9TccE92Wg9yGawXAa4-CIDAqCVCkJ3Q0uY_zCO0nSSB_zhpcEQRW4D6DyQkW3529fiIQcHCtOocHcWY-sW7IRnUAsW4yVq9yAqcMeCZ_74AuLOAvBTAqoFNdcnHgSfOHidT0KFml2D7Fm7BtozR62gkzIOfcqlce8a5p6j_umAVpL-FCg_I0mbPaTDmOdf5tPAvbA-JAafMhVLpb-UU7DUPzIMSd5PohhC1U87RTaTTew0-ToTWZrVVms_Fwkvxsb_O5IUjG9gp8aRgG_-_xK8XBRUk7j8rL3xqxeq0S7_Op9M06wwgIVi3_fTWVKc8XwMyQOwA-bhUxch36emi0PfoRYKAhFSxgNf2JgxwnYU6rXg8QRi2aH-KMMA94EPTYtajDmIcEJsFgIJQbAjADsMSWhRYtLAtgKBPD23Qx77uVLa82Q6K4xEKUo-Tr9p_d8xvDSNBU9KIb1YhZW5wKdkNvFkH1r_V_bzVyEnUe5_8mZhKLv8Mv5n6hXF8cnJ8eG78w_HJcqNUEm4kc-8XP_5086HSHxTqceRKHgU51L4ccynuQAxhzzOgV4EfyqYGzA-v-ihpqDOoqKKbO20FgDdlI7vRVYAF1Ci-lkkOuc1f75mUnpAWYpKH9JFTcaybkK6cpBZmHZOHZJSYbebinzB0MmOfNiDC2rO486Ry6drJFGADYH38KMw8qUT7l9yaXEaqkcuvg8ezH_oWe-y_ummoZKDLgL6YTeyDvBd2OXTHqmq5KJTqvppN_qR8Hkn7_phN_I83qWJelnWTmUO1UPpDaQ4UsnBaPajWAaJzpU9jsUcpJvg22IsMl0BdVK6BM9TgfQaThZ5s6LbjGsQPbJSaVKHhOSzblTdGOzgQT7r8USNDKhDoHWYblK639bpyPpQq25Nh_j00x4mqo5Vx_zLx90EdOHcuX5lFd5n35AXdNu3fNiNrO1jlYTOLoZQULp8JEIBXqhXmecwPfauQsn-XRoFuqnzKGpqZu3EFuxtREtbdiuzG1NUTa9HcrjcC3sS8ZfvXjqcbg1kDSHwlmc98XgYpXmb6AwSSFlvpvHjCK68Y3oUn2mR8yQ4ksXYYEZfvagS2FcqD8ghsc-KUzHJDx6gwj2Rnm6p5p3wAoKUmFzIfS_v0nn2fKH3wWjd40pOZTZZ6PqoTriCg_ycz2L5_S8zHkfFA5S6Zv3rkqr8UibVUr80HZmx5BAO315f7l_oh_txUVKR41R8a4GVM1FCQOq78ku1lWZ_MV0lDzXhKPHjWSDOk7J1NxdkrEsbuU1Ilkpzg8vjVLYiBJ_C0uCdZcrzgf_xCuZXIp_GqcdjvKCt-UX2TgMAzceQlM5VxgEyh8qP64b7V824fqKEt2Pcion8fAkpfwCV9IvzAsTyQsrmhZoQ_NVORVbiy8ypakvJRRYj8k3B8hxePB1zuhOvHTDknFuz10VnKIO_g6GnRfawAW_OOt7OZHt3a46a2EP50FiaiVea1vc0gTMBsal4nAGsleNhmtyl_uPUv43GUJnWcf9ekr1M40IkN2Owna0l20ZjqGQ1rqGRn9DyYP6RzL4OeJLIXsvWPrOVzHCnqdGNEv8p3WYSTUWWX_Lsy9bcLd41QSD7kkAuV-MQD2NRz-M_IQ7m_8oNNZ0fZiFl-EXb2YgSZdm5bk5n-CKoF3LXUKicprGYrNJU5LYNaMvEN16a8nWh7HQbqxNcvI9UEBrgKTVU529q563sX37ZIhLOU8A2KoN9TA33xY9Qiw-cx1q2tVe1T6karQNsqh6SiLH6SrmhIdc8grKievrjLBskeycL3e8a-xRzUhZFtn2S1kpkqGYqZKPCfsLwVxpOuXX75rHW105ncPaksY2boSa4KZv7QSCC-RaK6k3F1tz2khvKtCJS29dRfz3ytP5VmerbMBRJLu2k0E2nrbk_EUIeetG7MzZmu8Q2SvQf5jAO0rR4pL9YW4TEPANJbF-ALOEPNp8Sb7DqrDWbD1GhZrB9_FkmMJSXOeITOoGTWORjqf_yFenWLLVTGaz2EtuYo2-o9rhN7fVg67Q-eDCq9wgruo97dL8mw9VchzSelF82UwG9cW2-SouUY75K9XRqg6SIfKek6HvlRKUgnj6o6eEeHcxayQzOnBXy94tdx7Ha-sLjk9Sf5Y_gsoPQUD7n6IbGf3pOL6C4Fll5ClEsCu_HML2e5uBaQJMyKlpGndh3Kgl9Lvf4Pq4gbNAYutiqUWwc--P0R6YnfY76KYuakxgYPeL5-BGRcJnC8CgIfxoK9Uf3gLBx801u__6usi21T8eH03GaF2-T19E2ndO5GnfTGqzOOlpdjTk4sHzy95T8fhJNeCEqC9pSs9upDK7oNLZxOuNZEPHkx_c5f6COlzmZ3m_-yOSjSWXD3KO29-5JGdQynQf-R-dZ_fQ2T0Q0Ab2JVu9SzL-DFE5U9qrpP6Zn0EZmcNRQyEtM_gCrupZ9CvSdzelI5MLP-CPeAq9SGP72qcJckWn5540o1MYQvR9E_vkuE6La0aGIoHJTBXyo31Oit2yD7OTaPZT7BX_a-ZNgTNEutRnFf-3-ySzT3iWUOAQ-EJO4ZBc5xETwybKoi3YZMl0bPiHiMLyLMEKuhKSWBZ-QjSUetZFr71oM_iufuRSxXYosSiUeRtjaJQ615CdKAGjXspHt_gWCSAO5a8QmjrNLXAemZVNKd5GLTLIL6ABKGCN0FyN4CmgYA2FK8K7NNApydqkDo-9iRh2YrGm65i62meXuWi6xgBeTkF2KbdcCdJsAuostukuwC1QIRTAcDGE7cgjLAf5gZEYth-zCWEAdIYrNXUSQbe5SYgNjzHJgDGaCeKQ4KFHfOPJfOSdiUQuYx4zAICaGWTMTu7tazsS24CFwZAJ7CMSCHNuCoeUEYD0QzN1xYO7EUpNxGbCgheu4FOZou5IUYpZcJZvB36YNa0SZ6eJdC7kYxKZXzaQwBsYmrJleVb1ihDlAmDIHSMJwIBvCTAQzw7YcnGFgE2QtV9OCVdq1QFewFDFMEMCRmrYL_FEbFsECYVHiSgUC2pbpwPJgWGLQDKUtICRYQmZJoSICawMjOIztIgBFUuI2iMfBUsygc-6uS4ANbEvJ2JZLAV9qBgWVk8KXmoNhZZBcTwYShEnDVFwYBZAQCBy-RmhXK7jWQgRrLleeytWQ8oV1gPkymL9UbYrkyiAlTmSr9QRlhWFhPYnNbDDWAsytdoeQ9Cf-wry01b6_vtD-aFwU0_zlixffvn3bm_JinIbiHiq0PT-dvJgCEtjsc-V4nkuyL_bhfwe3l0fe9R_-hB0HJ8dfw8_30w_0zfPXxx_2xeHPs9d3X9zEF1-Zt5__emH-fj99uBtHlweT29c3J7-du6cf_N-O80_3Zx9P0jw5evPwNX-XR5F7d_ttMr4g5m-4MH_n9Morfvnlw8_k4ejDa5Bdcmh_Pvv9MrfHwfnddXEaHn24-XBqHl1m7_DlVX568tssyCanz88_seKPk5vXvz3ch87vk-QLtvD11eXdBXt9sB---8zO98_eO9cXvyDnPp2Grz__NrHPr57nf1xc86tzxj78-pHi6dn4BtKiez_4Orv7kr7lX0P8Lpl--_zLmXN2wsdFhgm5n4Tjk-v73-_C4OL5Tbgf2s_vnt--955fIJLZ08tk_9tJtP8xPfht_-vpu_s4Iz8L-qtgU3z55iz8lVsX_uz0-aH765vICz-y6afZeOo6_OfzSRh9fBO5gXav1Uq90qeF8jKM6E_KH6njSNIBQNQqtO8naB7XlwCVd6oB2l2AyoHVALvglKOpwdEuwPLEVAXYOUPliRZweBFoFgKADxA7VJiRwUX-8SYthHomv6w-qNAjz66Up6tuikyG7j_SdPIbTIERtkdl7FAWpTcBngleXPLpPCpL2HITIULlJsKCF0eR7MuqTahVuJSQvwKUi_YYNZEFdsxsfeTulYrBZQyUf1ebGme50DfS6FRPfV3bdihB61sOr1XX_qXx_s35L--PR0eCB8a18L-k8nXQ6FCmC4XxszwCNXqfRF9nwjg_emnYgUkdiCU2IiRAphf4oRfCstkWQvJ0EnZCzwM_zVEIvpET8IqmjzwHQgunbiBGahpKCEDMGl3zIJrlL41LAf-djNRxVhEYRfrSQKPzyTSO_KiA5-ZI59u5MeZ3wniGzH8CkMFlp2Sek17Pj4flI3kK0-AFZEKQAhjUNM6TQsRxdCvlDCmZoUfeXemmGj5Pqi9nUzkIshYHoi8h6c2V1khGmoLFTcFe718fj07jh-nYOH4Qow9RFsmybFWmbsiFZXLX82zhEyuknEBAJ4HNWYggQCDqY0_GbN8KcejggDgYi4CabgiYptWQqYObYnvGKjktDjyM7H_KpDQD0YCsJ3rnryHPM4yeIaQEKw_6GzwJDHkgvyHYdt5JC-8HGZ94qlkX5bC08oCzcRjPVDa9IgQIxA6HUEXBYn0ndC2L2K4fQPgTngkK5CHhQHwGpXMs37eE6_gcHjgYZMddjJtCICP117X4-hLWvi4RMvoTcs4QlOyv_SDIDce40qHJ0Enm4jE29DvS5nODg2DU7I3SiTQIysPzEyntJazbjCcFqHmpxIEAdZU6vFgGXVCNemiALacVoWdYrtJ-c1lR24yNKDcOJTWAL8bwZcxnt2NYg27wE2kz6i1pBPl0D91rkYhvPO7SCdSiFIdQRvjxQz4xDmdFp5shIQJnilwPrCCgrvCYFQYkDEzPcTEoA2R-iFuB6wgfM5t6rhv4VDgMBeCFAmY3tcGsaQNmdW1AMIMsm02lzzmI0zSQ1p-kheFJHxECmHwCdv-QzkbSNJbXy5Dn-wx5wG-EWP2xLriM6laSanmbCsBaFKBybXMLXLyjr5vhfN5dwqdt3miczmKYGlARyegwiyZ52uKQAt9DhAqfYsGwS0ObYOHBkjiUQmILSxOGgc-xy3xuo8BDDsDjkPhYQDHjc9SU_pIfB_4kexA-RXIrfTSweCTuQSFhniNk1UWiO2G97upphGW3xsTD6lBwtX3wCnKBxH_oVFufWVxA4YIIZxh5oQP1ViBCE_sI9DRgoWlBXi0C1xYipLZNfdMBZwZeL7Q9jpvREfVFQ1oXiS5vjTSUapqBYWU5MH0iT1eD15jKayUlEflUh09nFXtE6t-JOcU3afJcETTkrvXcCLN0oocp_WW7RNu07yjjt6BuJ-BTRvuQQcHfr4E_Qx9iG-nFBgulbvn3AcylvMcK6ghge485xB7po28l2kvDwqP6YbhVJNvGtBFvfddzuM-pbRHLsx07sIQPUYRavmDCJKDuFqQ1ghFb-iDLC4iAgBx63OU2tcLRzZhPweols01Vt0bloQ_gwhyV8QG4f37w_LTmg5jT9EHPEF6TzWhS0q3I3E-tIWlEj2t17ZfaAS4XFEgrRUWgJ_ogrKSvjqwZ-v1S6W1wi70pQ2oIefQMW5JA0_YIUbNuM7klX3lTzBJFVc-guqGrQ3esVmO8-TKL4zFkqSPIcmdFmhlyA_pCa0yzU2scQkbleUMNWH5oh6xrigVpfUC4R30eCCt0zNCGXBdyEMIDG-p-l4Pa-K4bWoEZOBQ5EK5CbJOQ-Y5rNcOQxRbaUVOOA1CP64Z6ENa0ddvuWiPNBuR45urqMKf-nTxxCnrmrNGz1SR7WXt0G3Q5nGl4glSyWXrt6pm6kkPIPp8Bag4uRc7PkAaUq3KlQwuWXPLl_un54ehwLCYw2cof38zi6RiEoVydJC0THIARBR83qhZhe8IFMzcFEWZoyQ5K6JuE2Zxi07ZoYHmm7Lv5PvXAR3vYJzYkn0QEtilC0mfl3WZ9CCsGaRmk0EnZ6AT7zdJZojKK9zlYH6xHJm8qgyflDUBKMFDKjVDDIVe3Qen3DUYwy1R-pvjWzrpDjM4AMf4yi_wveRTLVKYhyrffGuHN8lzkCALyg3oEgZ-mDKyJQMQ3hUdd1zO575jwyfGhRHEww4HJpZwDhAVaCm_dYrSb6o_tbiExtHi2iII6AJZpW1NWx32ycoeoXIuYDgTP6nKiFoXE1DRD0yN2yKzAR6YNAQfih8sdQR3ThKBv04CHkF4JBJHFh6oOymkhUwJzoJysJTn1KBMzW-U0lnl8j5jq4j3JhPhDGEeQBagos7F0kdkm3ndQz-S38pzuIs06iu6iRGg_0RD0m_QuguBSz7hAAcF2wXBdL3A9Rm3BoIT0PBSEjo88KyC-hz0_wDTEjudS3-MMKk3MXGqBS2-Impl1D12TtNmUtDwcbtC5kL-NhZaFLA3PIigeHsA9ywg6eRidynothyRl4fqq8Cd9Zekku0TWVkEdzUAgH8dpFo_efUuf6-MT8mVRw1RtX1qdZ_pMXhsjPMwZuGfLtAJqmiAJFIZgtcL0Qo8RGoRMuLJ4coUF5kpos0-Da8Kw8HLyYq3pGDyjqJ7xSwz5ud6QARhnNYRhthrCsE741dV69YSjQ4BtPZnLNPMi6YqjKYRhCP8ZWIne0b-cWyKwuDXJpQVhvl65IswdKI-Y49igfI7rCw7qZ7umacLf2LZcgSDFNBkPuElCKAMCWCjmYRceW0vNHLs9oTyFhBL-X1dRd6nKokSKaVFYuc2cfym7a8kViL1Y1pqcn2Gdqy6tcpf82_pCZ1CNQfp2obgZvU0e7o39ySyGv2ty9AgEXk84oYMFxQELwpBgVxBMIcj4IFiHmiEYPwuwJV9_cMps4lnwZeiadsCWDbvWD1rJvklLi6zMeBdKS-ymQJvxuZLdkmAd3JI2O91pc4cUaWtGvB97UJwmldf8yDP51gW0JA4i0Gi15XyeHxPb7suPybLmQ8nUp_iovlZIOIhA7AqoA6vBGPV9yJpNzjzTCRwb-y7irpCZluML33UcU3YRGCaeh0kgvOZakXadP3h-vaTz1G0u5LwP12ysLaVOUTivg0vRyW4PeHDpucE1A2S83IeYa4FqGSpv5pilDyr7QDLv1TsEQHWQXcuDq-puxWXhnmkCVl1z9FVcMoLM-9RdumK1WNzHcZRP1dZv4yrNChhldDrmOTAqW9MtXX6Ik75NZdlrIZN7YeDhwIeyhnIHamLmCJP6zLUsMDjPwqZvh1i4IYNFJT4OmwkMteq1zXI3CJMVC8Hsn4Y_LzvURSWwQvqEzixJpHDwmsKkWuVv4NvLrwIVlpur3OjGKVPFyOBBEMkMB75aPh7TJXPWFqaFPtyYcYgyxcOtbP1_iPJG9hJQxi3ETEeEkCxjmakE1ONgTdgPXR56rpBVClSekK5gW-aNjIaWY2IPShfGm8GCLYxlv2kZZ1xq7b73kOe86luPnlnukodDrZ69pZSvJ4EyzYnAYV6LUF7VloMcl5qeKs-Rd0yXaySy3uaUNJeV7tTctp9ZdEVhOlalvamnjkJoatmo9kM2MrP_spIAEGqvSQAsCn6otqamxWBZbSx8ucsCh1wICu6PgKNzLZN6tiscS94OCAEtEDJD8yBmYUpkReA5TduxUZczlP-HNMCoO0RrKQlAxOoO-pXuy7sDRFaUtTlk2VFmXNakq1avEd1GmoIqLtA_V_yapPGglnLU9JDyphcjSCFjVh32B7knoEqAtSfumKLuWcIEQ70VC8bJVAHTAJdmrTJy6e5iYTSwG5C7JZvyGh7j_DZJwUlcAqik3d4X6dAwp1XDrmeTqArKIF0_VbfiqEpAtjuV6jdydYQ42L4N4RH5YNaB5RNfMALFtePawnMZheTd49j1Axa6UMo4ofB8h9qEWS5fedkxpDth6jd9_U60rcvwjPV3FDdoTaClelvfnzN6C_KHZCYJ6jIKOYZUkEiTgXoFcmcK6aAQTIDhEAG1XWh5juAAYXtgdSY2hefYsl1sYURps55hdnsnDszpum5OpOlFSdMR1tuoHS-EzZa4sD9NwTU-TPXB2jgenfDZPIeD1Dips41FEHiCI9P2ncCG4MsZdTjxIQYwDq4kgGKDMosyAokUVL2hjS1OkeMSYTNkNQME6WJ7qf1oNdnGpJPtpquWN0kqt68P0pZqAnFF-5FbLu9tMlTXH9hu9q8PZlk9zjYDc8t7ty6JtxXMb5Ncv_MEa4xknpOLYqVgNj2ZnILcuOV4FuSmAtIcP0Q2tyA8C0iBTIqxbQaQvpLAZLbNwtCnIfhsJLBvL7_zWtRqdY1qC8tVOEaNanepkGCr_fdaXxWxPWulrfqM0tZ4XnWBV_JRc4-oGDAP301voG6EhAWASfdEXdxWcY9nYwjus5G6p-aLXGdl34c6pr805KaX0VUmwuj-pfEn-Mlb8RIi61_6JapSOfniQwkSrQD-VT9D9jbR218WYPJ35kY3s7COg_5a-gkAugwhhz-vlE4eolNp-QJMUe2OzbVFt5caJoeQCGYpGFG5g_wyDaIwkkkRchvmJDeu1IsBRVIukCxVtR3qN_FSNpATu9115fp9B6ureQcryMtNu2pl29_LLN-Bbfy3jei_DHeP_Gt-8_X_jKprtMta40Lw2xmkTxfiVobGD3ooyFv1Fc2LL97wPJrwxecDHoNG8JG-FximP3-CVseYv_M35_uM1C6Uvherf5Zsg4ocCchTAEq-WYsg3oHkgTWQ4Rd5z5zcQyR7klx_gofV5Ock8MYkSnbnFMjGFCoBlXs1cgmRKeZVe7J8pa2rIYl5WV62PToDXUmzyB-iCWygJjCLEKkK1t9QFSRvj9WFfhqDlKGfxA_RBnuoNthKG_6OumA_gS7Yj9cF-_9cF5yhMYI6zt9TFyRnj44RvTSGBYleEj9EF9yBumA5CP09dUFy9lhd6KcxSBf6SfwIXSDmQF2QPY2_py5Izh6rC_00BulCP4kfogt4qC4QF_yC87fMHiVvj9aGXhrDtKGXxBNrw02cVj8fs7gZc342p_p5JwWlT7dUp2E0ju4o1o7zOHUMDVPdyzc_GYRXYQ6EPNk6h2DDxrUWKPYqzeou3ArEGka0xv_8p9uWZrr01qaBMnCU2tErdxgGrg3SIuXFteIVFO0GKi8FmNNzO0HREihuEYk6FF2fHmoRW_0ykzmg3UGsroOtxNTFbXMyLZzqzQs1mOqX-8rTWT0ns5p3wK9KRApkWQNqmmhWB9rahLmEhoehrVrkqji2nFEbM2tm1WYBnSibOI1WVW2bINlO2sNEgp8IjW43SbbdaAPR2qSJ1ix3m3atWQHtejcdqEWv2DbjrNHfrng0gDLZ3qjQNpzQ7ZQBbad6dP36rMboFtFuSrdNwbbkfAs_tJpErJ_dFh6SrBPCPD5uuJrtMt9K3axtkAaZ6EoWtk0aNoBta2sp4y2UlG1nZ2Q73SabKx3dPipsoeItafBQ6ZPN7XY1g94mhV7L1CPyqIHrvJKxD14jtoXOoq0t5BHKNNSbt5clm9Qlgxdnjey6apoBPghvLeEt_cnwJV0tvQbwQwZIqqVgG7wSdGt5WdtExiFGs1pbDlb1NXPqrEtflIWpuvREXSeifj44TcLoduWXfmXdqvbDqv0f6vLpdT8jDPWuvl-9PCGzHkPIszLn-UEqf-Kl-jFh9elF94Te5yKo3c1c7epdP5rHJYXFQHIvR9tAPMuAuPrhkTeCZ95D9TvK1S8juy1YU54UY5EmAJtmp2mwdpiwfqF943Z7fevben70j27n7zcWe4l4WElUXlCnf_J0HeacS_65m8urmPtinMaByOrrXN63OP95adPsh1cblcqf-5xjYasfaeX3ESpEhMw1qHIfW_nrwXOsNSjylMzyDOk6tuCbzScnR9ocS7WwFwgOWS-B5UHIkFVaRsI221u3UlIOm8qu2qK2hSSa-75Whnz1ovKDr15c8WL8NlS_-A4I_x79L6r_6gI=
```

**Comment by QuickStick123** [comment]
This seems to work for me with the provided pob.
https://xanthics.github.io/PoE_Weighted_Search/?vals=6,13819.0,13819.0,949.2,,12743.3,11307.2,,,,,,11307.2,10634.1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,5548.2,,911.5,2656.6,,3808.0,31.4,2562.1,1557.6,1905.0,216.3,,,,,,,,,,,,,,,,253.9,,,2284.8,,,,,4249.3,,5853.1,7977.0&FrenzyCount=3&Flags=Leeching,Multicastable,enemyIsBoss,Spell,Totemable,feedingFrenzyFeedingFrenzyActive,Mineable,Duration,UsedMinionSkillRecently,raiseSpectreEnableBuffs,minionsConditionFullLife,minionsUseFrenzyCharges,CreatesMinion,raiseSpectreEnableCurses,Trappable,Cooldown,Minion,Shield,&Skill=Summon%20Skeletons&Character=Test
Was the generated dps weights.
The way I got it to work was dequiping the item to check and following
![image](https://user-images.githubusercontent.com/31533893/180126375-1f9a1e3c-d39d-4ff8-a2b1-de7b75728d67.png)
These tips are probably also important.
![image](https://user-images.githubusercontent.com/31533893/180126455-ed9bdb6c-aa17-467d-91c2-739bfb15d5eb.png)
![image](https://user-images.githubusercontent.com/31533893/180126676-3dc4602c-c588-4d75-8523-007a3dcfbb69.png)
Closing as invalid.
